packages/media-query/src/use-breakpoint-value.ts

```diff
@@ -11,7 +12,21 @@ import { isArray, arrayToObjectNotation } from "@chakra-ui/utils"
  */
 export function useBreakpointValue<T = any>(values: Record<string, T> | T[]) {
   const breakpoint = useBreakpoint()
-  if (!breakpoint) return
-  const obj = isArray(values) ? arrayToObjectNotation(values) : values
-  return getClosestValue(obj, breakpoint)
+  const { breakpoints } = useTheme()
+
+  if (!breakpoint) {
+    return
+  }
+
+  const obj = isArray(values)
+    ? Object.fromEntries(
```

Comment:
necessary to make the resulting object of `arrayToObjectNotation` compatible to `getClosestValue`